addons/web/static/lib/py.js/lib/py.js

```diff
@@ -969,6 +969,15 @@ var py = {};
         __init__: function () {
             this._values = [];
         },
+        __len__: function () {
+            return this._values.length;
+        },
+        __nonzero__: function () {
+            var len = this.__len__();
+            if (len === 0) return py.False;
+            if (len === 1 && py.PY_not(this._values[0])) return py.False;
```

Comment:
Not in my python shell (altough maybe for some obscure immutability related reason, I don't understand why it wouldn't) : 
```
17:54 $ python --version
Python 2.7.8
17:54 $ python
Python 2.7.8 (default, Oct 20 2014, 15:05:19) 
[GCC 4.9.1] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> bool([False])
True
>>> bool((False))
False
```